src/Rector/MethodCall/ResponseHelperCallToJsonResponseRector.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RectorLaravel\Rector\MethodCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Expr\New_;
+use PhpParser\Node\Name\FullyQualified;
+use Rector\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @changelog https://laravel.com/docs/7.x/database-testing#creating-models
+ * @changelog https://laravel.com/docs/8.x/database-testing#instantiating-models
```

Comment:
Thanks for the PR :)

These two links are not related to the rule 😅

Reply:
Sorry for that, typical copy + paste mistake 😅